[AGNTLOG-229 Multiline features FAQ (#29931)
DataDog/documentation: content/en/agent/logs/advanced_log_collection.md

**Note**: The Datadog Agent does not start the log collector if there is a format issue in the global processing rules. Run the Agent's [status subcommand][6] to troubleshoot any issues.

## Multi-line log aggregation FAQ

**1. When should I use manual multi-line rules vs. automatic multi-line detection?**

> If you know the format of your logs, you should use manual multi-line rules for more precise control. 
> If you are sending lots of multi-line logs and you are unsure of their format or don't have the means to configure all sources individually, you should use automatic multi-line detection.

Reviewer: ```suggestion
If you know the format of your logs, you should use manual multi-line rules for precise control. 
If you are sending lots of multi-line logs, and you are unsure of their format or don't have the means to configure all sources individually, you should use automatic multi-line detection.
```

**2. What happens when a multi-line pattern doesn't match any logs?**

> All non-JSON log lines will be processed individually as separate log entries.
> All JSON-formatted log lines will be treated as a single line of logs, and only the first valid JSON format will enter the intake; the rest will be dropped.

Reviewer: ```suggestion
All non-JSON log lines are processed individually as separate log entries.
All JSON-formatted log lines are treated as a single line of logs, and only the first valid JSON format enters the intake; the rest are dropped.
```
